src/custom-types.json

```diff
@@ -831,6 +831,18 @@
       "https://en.wikipedia.org/wiki/Processing_(programming_language)"
     ]
   },
+  "text/x-python-script": {
+    "extensions": ["py"],
+    "sources": [
+      "http://www.htmlcodes.ws/reference/mime-types-complete-list.cfm"
```

Comment:
This URL does not contain a reference for the MIME type entered `text/x-python-script`.

Reply:
text/x-python-script really cannot get the reference, but not sure why server was returning this. we manually handle this.